src/Styles/style.ts

```diff
@@ -0,0 +1,39 @@
+import styled from "@emotion/styled";
+import { Padding } from "@mui/icons-material";
+import { AppBar, Button, Card } from "@mui/material";
+
+export const StyledButtonHollow = styled(Button)({
+  border: "2px solid",
+  borderRadius: "30px",
+  borderColor: "#EADDFF",
+  marginRight: "5px",
+  padding: "5px",
+});
+
+export const StyledButtonFilled = styled(Button)({
+  backgroundColor: "#6750A4",
+  color: "#FFFFFF",
+  borderRadius: "30px",
+  marginRight: "5px",
+  padding: "10px",
+  "&:hover": {
+    backgroundColor: "#FFFFFF",
+    color: "#6750A4",
+    border: "2px solid",
+    borderRadius: "30px",
+    borderColor: "#EADDFF",
+  },
+});
```

Comment:
The previous buttons were fine. This was not needed.
Only use it when you have a very custom design.
In our requirement, we only needed to change the colors which was handled by the theme itself.

Also, don't name them 'StyledButton'. Use 'Button' or if it's a custom button then name it according to its use case.

Reply:
Okay Atmeshwar. I used this Buttons as I had to use sx props everywhere I need Button from mui.I will change the name of these Buttons accordingly